src/inspect_ai/util/_sandbox/docker/docker.py

```diff
@@ -173,6 +173,7 @@ async def exec(
         cwd: str | None = None,
         env: dict[str, str] = {},
         timeout: int | None = None,
+        user: str | None = None,
```

Comment:
I'd say let's move `user` above `timeout` so its alongside the other "execution context" parameters.

Reply:
Happy to do this. Though this might be a breaking change if people are using positional (unnamed) arguments. Are you happy with that?

Reply:
Thanks, I'm not worried about the positional breakage here (as I think specifying all of these params for an `exec()` call is almost certainly not happening in the wild).